### ISSUE TITLE
[Visual Requirements – Make Code – Help] 'Help' icon control is getting disappear after increasing the zoom level at 400%.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3610]
**User Experience:**
End user will find difficulty to interact with controls and won't be able to perform functionality over such control if controls are not visible after resizing to 400% due to this, user will be affected.

**Note:** User credentials should NOT be included in the bug.

**Re-pro  Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Help" option appearing in header section and select it.
6. Navigate through all the controls of appearing after "Help".
5. Go to browser setting page Zoom up to 400%.
5. Verify the issue.
 
**Actual Result:**
'Help' icon control is getting disappear after increasing the zoom level at 400%, due to this, user is unable to interact with controls and won't be able to perform functionality over such controls.
 
**Expected Result:**
'Help' icon control should be visible after increasing the zoom level at 400%, so that user could easily interact with controls and could be able to perform functionality over such controls.

**Note:**
1. Same issue exists on Chrome & Firefox.
2. Same issue exists under 'Pair device' pop-up.

**WCAG Reference:**
[WCAG 1.4.10 - Reflow
](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.10, #Win10-Edge(Chromium)

[WCAG1.4.10_Help.zip](https://github.com/microsoft/pxt-microbit/files/5499741/WCAG1.4.10_Help.zip)
